src/NuGet.Config

```diff
@@ -5,6 +5,7 @@
   </packageRestore>
   <packageSources>
     <clear/>
+    <add key="local package source" value="d:\packages" />
```

Comment:
@stephentoub @FiveTimesTheFun I believe this line is causing all of the dnx hangs.